### DISCUSSION TITLE
Filecoin Ecosystem Grid

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#194] [General]
🌟 Are you curious about what are the existing application and clients, collaborators, capitals, accelerators, dev toolings, infrastructure, R&D collaborators, and so on in the Filecoin ecosystem? Take a look at the grid below!

🤚 Do you know any other projects and parties are building in the Filecoin ecosystem but not included in the grid yet?  Leave a comment in the[ Filecoin Ecosystem Collective](https://github.com/filecoin-project/community/discussions/195)!

**Comment by jennijuju** [comment]
# 2021-06-30 Filecoin Ecosystem Grid

![image](https://user-images.githubusercontent.com/42981373/123996033-e8df3580-d99c-11eb-94ca-4176c5e7759b.png)

**Reply by jimmylee** [reply]
Wow this is incredible. There are so many awesome forces in this ecosystem.

**Reply by whyrusleeping** [reply]
@jimmylee we should put estuary on here, we dont really have a logo, but maybe a simple typeface logo would do for now?

**Comment by jennijuju** [comment]


**Comment by jennijuju** [comment]
# 2021-07-13 Filecoin Ecosystem Grid

<img width="4903" alt="filecoin-ecosystem-grid" src="https://user-images.githubusercontent.com/42981373/125494283-3da96027-7faf-42c8-aa81-06853e2c0170.png">

**Comment by jennijuju** [comment]
# 2021-07-27 Filecoin Ecosystem Grid

![filecoinecogrid_v07272021](https://user-images.githubusercontent.com/42981373/128069491-7442d297-4e4a-47e9-8b03-fdf1a7eccdb1.png)
